docusaurus/docs/migration-guides/angular-react/suggestion-supplier.md

```diff
@@ -0,0 +1,78 @@
+---
+id: add-suggestion-supplier
+title: Add a suggestion supplier
+sidebar_position: 9
+description: How to add a suggestion supplier to a plugin.
+keywords:
+  - grafana
+  - plugins
+  - plugin
+  - React
+  - ReactJS
+  - Angular
+  - migration
+  - suggestion
+---
+
+# Angular to React: Add a suggestion supplier
+
+You can add a suggestion supplier to examine query data coming from a panel and suggest usage of the plugin for the type of data detected. This guide provides instructions for doing so along with links to relevant examples.
+
+A single stat panel would be ranked "high" for a single series, and "low" for multiple series (or even none).
```

Comment:
This sentence seems disconnected from what came before. Can you clarify?

Reply:
will add text here, i left out context